project-koku/koku
# Add fallback to effective cost calc in Trino

Fixes [COST-2284](https://issues.redhat.com/browse/COST-2284)

Changes proposed in this PR:
* Add fallback calculation using fields that existed before effective cost was calculated
*

Testing Instructions:
1. Comment out part of code that populates effective cost in ocp daily table
2. Run standard ingest flow and check for value for effective cost in daily summary table
